src/components/Pagination/Pagination.tsx

```diff
@@ -61,7 +61,7 @@ export default function Pagination(props: PaginationProps) {
 
     return (
```

Comment:
The dropdown in the pagination story has broken

Reply:
I imagine this was broken with #253 originally. Because this will require reworking the component I'm going to request we file a new ticket for the work instead of trying to do it here (#220 is blocked on this work, which Location Finder needs more immediately).